packages/fast-components-react-msft/src/action-toggle/action-toggle.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Action toggle",
+    "description": "An action toggle component's schema definition.",
+    "type": "object",
+    "id": "action-toggle",
+    "properties": {
+        "disabled": {
+            "title": "Disabled",
+            "type": "boolean"
+        },
+        "selected": {
+            "title": "Selected",
+            "type": "boolean"
+        },
+        "selectedContent": {
+            "title": "Selected text",
+            "type": "object"
```

Comment:
What does this look like in the editor?

Reply:
Changed to string which works and is editable but does not enable the full range of options for this prop which takes a ReactNode